### PR TITLE
Update 360

### PR DESCRIPTION
**Pull request by nroduit**
Update to Weasis 3.6.0.

I skip the test to build the package because there is an issue with the test class.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -50,13 +53,13 @@
         <dependency>
             <groupId>org.weasis.core</groupId>
             <artifactId>weasis-core-api</artifactId>
-            <version>${weasis.version}</version>
+            <version>${project.version}</version>
```
Comment (poojavenkatram) [inline pom.xml:56]: Where is this value coming from?

**Comment by arvgord** [comment]
Hi Nicolas. Could you add my [PR](https://github.com/nroduit/weasis-chcapi-extension/pull/1) to your PR to fix some issues, please?

**Comment by nroduit** [comment]
Yeah, I just wanted to give a hint on how to integrate the latest version. Unfortunately I didn't have time to look at the problems related to testing or compiling through Google tools.

**Comment by arvgord** [comment]
Thanks! I'll create new PR.

**Comment by poojavenkatram** [comment]
Thanks for the hint Nicolas  - this has been integrated and fixed in [PR#39](https://github.com/GoogleCloudPlatform/weasis-chcapi-extension/pull/39). So closing this one.

**Pull request closed without merging**